test/greet-test.js

```diff
@@ -0,0 +1,14 @@
+var expect = require('chai').expect;
```

Comment:
Tests are solid for the current state, but the file/dir structure needs to be updated slightly, which will mean a little refactor here as well.